.github/workflows/nightly.yml

```
@@ -93,10 +94,28 @@ jobs:
         echo 'Architecture: all' >> output/DEBIAN/control
         echo "Maintainer: 鸭鸭「カモ」 <${MAINTAINER_EMAIL}>" >> output/DEBIAN/control
         echo 'Description: GHV Application' >> output/DEBIAN/control
+
+        # 创建安装目录结构
+        mkdir -p output/usr/local/bin
+        mkdir -p output/usr/local/lib/ghv
+        mkdir -p output/etc/ghv
+
+        # 整理文件
```

```suggestion
        # 整理文件
        mkdir -p output/usr/local/bin/
        mkdir -p output/etc/ghv/
        mkdir -p output/usr/local/lib/ghv/
        mkdir -p output/usr/local/share/ghv/
```

https://github.com/fjwxzde/GitHubView/actions/runs/12849823631/job/35828824337#step:12:90